R/line_length_linter.R

```diff
@@ -4,10 +4,14 @@
 line_length_linter <- function(length) {
   function(source_file) {
 
-    lapply(names(source_file$lines)[vapply(source_file$lines, nchar, integer(1)) > length],
+    # Only go over complete file
+    if (is.null(source_file$file_lines)) return(list())
+
+    lapply(which(vapply(source_file$file_lines, nchar, integer(1)) > length),
```

Comment:
Could you split this statement for clarity `oversized_lines <- which(vapply(...)); lapply(oversized_lines, function(line_number) {...})`